doc/gbs_tutorials.rst

```diff
@@ -4,4 +4,9 @@ Applications layer tutorials
 .. customgalleryitem::
     :tooltip: To be filled
     :description: :doc:`tutorials_gbs/run_tutorial_example`
-    :figure: ./_static/planted.png
\ No newline at end of file
+    :figure: ./_static/planted.png
+
+.. customgalleryitem::
+    :tooltip: Point processes
+    :description: :doc:`tutorials_gbs/run_tutorial_points`
+
```

Comment:
To continue the in person discussion, if you could decide on a nice screenshot we can add it as a thumbnail to the gallery page